packages/common/src/pipes/date_pipe.ts

```diff
@@ -207,13 +218,14 @@ export class DatePipe implements PipeTransform {
       value: Date|string|number|null|undefined, format?: string, timezone?: string,
       locale?: string): string|null;
   transform(
-      value: Date|string|number|null|undefined, format = 'mediumDate', timezone?: string,
+      value: Date|string|number|null|undefined, format?: string, timezone?: string,
       locale?: string): string|null {
     if (value == null || value === '' || value !== value) return null;
 
     try {
       return formatDate(
-          value, format, locale || this.locale, timezone ?? this.defaultTimezone ?? undefined);
+          value, format ?? this.defaultOptions?.dateFormat ?? 'mediumDate', locale || this.locale,
```

Comment:
It'd be great to move the `'mediumDate'` to a const (and store it in the `date_pipe_config.ts`):
```suggestion
          value, format ?? this.defaultOptions?.dateFormat ?? DEFAULT_DATE_FORMAT, locale || this.locale,
```